src/jarabe/frame/activitiestray.py

```diff
@@ -309,12 +310,34 @@ def __activity_added_cb(self, home_model, home_activity):
             group = None
 
         button = ActivityButton(home_activity, group)
+        #If its the journal, the activity name is None
+        if home_activity.get_activity_name() == None:
```

Comment:
Is there a better way to check if we are dragging onto the journal only and not any other activity?